olca-app/src/org/openlca/app/editors/graphical/GraphConfig.java

```diff
@@ -13,6 +13,7 @@ public class GraphConfig implements Copyable<GraphConfig> {
 	public boolean showFlowAmounts = true;
 	public boolean showElementaryFlows = false;
 	public boolean isRouted = true;
+	public boolean editMode = false;
```

Comment:
a better name would be good here. isProcessEditingEnabled would be better but is a bit long

Reply:
Maybe `isEditingEnabled` could be enough?